src/cpp/src/utils.cpp

```diff
@@ -266,6 +266,42 @@ ov::Core singleton_core() {
     return core;
 }
 
+void print_compiled_model_properties(ov::CompiledModel& compiled_Model, ov::Core& core) {
+    // Specify the name of the environment variable
+    const char* env_var_name = "OPENVINO_LOG_LEVEL";
+    const char* env_var_value = std::getenv(env_var_name);
+
+    // Check if the environment variable was found
+    if (env_var_value != nullptr && atoi(env_var_value) > 2) {
```

Comment:
https://github.com/openvinotoolkit/openvino/blob/b2bfd851a1568c7cb496780eec9af092e0935a1c/src/plugins/auto/src/utils/log.hpp#L111
Can we use ov::log::level instead of '2'?

Reply:
updated